README.rst

```diff
@@ -274,8 +280,8 @@ We can take the example in  `Using Estimators <#using-estimators>`__ , and use e
     # Serializes data and makes a prediction request to the local endpoint
     response = mxnet_predictor.predict(data)
 
-    # Tears down the endpoint container
-    mxnet_estimator.delete_endpoint()
+    # Tears down the endpoint and endpoint configuration
```

Comment:
this is still in regards to local mode. So it should state that it "Tear downs the endpoint container and deletes the corresponding endpoint configuration"